### PR TITLE
(DOCSP-23473): Updated Watch for Changes to Reflect Serverless Incompatibility

### PR DESCRIPTION
**Pull request by chloe-ratte**
## Pull Request Info

In the "Watch for Changes" sections of various SDK and other pages of Realm Documentation, added in an explanation that serverless instances -- because they don't support change streams -- are incompatible with any function to watch for changes.

### Jira

- https://jira.mongodb.org/browse/DOCSP-23473

### Staged Changes (Requires MongoDB Corp SSO)

- [various SDK pages](https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/DOCSP-23473/web/mongodb/)

### Review Guidelines

[REVIEWING.md](https://github.com/mongodb/docs-realm/blob/master/REVIEWING.md)

### CODE REVIEW

File: source/includes/severless-watch-note.rst [hunk 1]
```diff
@@ -0,0 +1,3 @@
+MongoDB severless currently does not support change streams. This means that
```
Comment (mongodben) [inline source/includes/severless-watch-note.rst:1]: add the `.. important:: Severless Limitations` as part of the include. nothing stopping you from doing this + less rewriting content

Comment (mongodben) [reply source/includes/severless-watch-note.rst:1]: also i'd put the fact that you can't watch in the first sentence. the fact that it's b/c of change stream is just an implementation detail. worth mentioning, but put in second sentence.

File: source/includes/severless-watch-note.rst [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+functions to watch or observe changes do not work if the configured data source is
```
Comment (mongodben) [inline source/includes/severless-watch-note.rst:2]: rephrase to not explicitly mention word "function". depending on SDK, this isn't nec correct. 

maybe rephrase to something like: "You cannot watch changes if the data source is an Atlas serverless instance."

File: source/includes/severless-watch-note.rst (hunk 1 above)
Comment (chloe-ratte) [reply source/includes/severless-watch-note.rst:1]: oh makes sense!!

File: docusaurus/sdk-sites/java/docs/app-services/mongodb/watch.mdx [hunk 3]
```diff
@@ -14,6 +14,10 @@ These code snippets demonstrate how to configure and run
 [watch](https://www.mongodb.com/docs/manual/reference/method/db.collection.watch/)
 operations on a collection.
 
+.. important:: Severless Limitations
```
Comment (mongodben) [inline docusaurus/sdk-sites/java/docs/app-services/mongodb/watch.mdx:17]: (and you also should remove these)

File: source/includes/severless-watch-note.rst [hunk 4]
```diff
@@ -0,0 +1,5 @@
+.. important:: Severless Limitations
+
+  You cannot watch for changes if the data source is an Atlas serverless instance. MongoDB 
+  severless currently does not support change streams, which are used on watched
```
Comment (mongodben) [inline source/includes/severless-watch-note.rst:4]:
```suggestion
  You cannot watch for changes if the data source is an Atlas serverless instance. 
  {+atlas+} serverless currently does not support change streams, which are used on watched
```

File: docusaurus/sdk-sites/java/docs/app-services/mongodb/watch.mdx (hunk 3 above)
Comment (chloe-ratte) [reply docusaurus/sdk-sites/java/docs/app-services/mongodb/watch.mdx:17]: the whole files yeah? i noticed they weren't .txt but i wasn't sure how they ended up in my edited files, probably i just didn't notice -- but basically i shouldn't edit these files at all yeah?

Comment (mongodben) [reply docusaurus/sdk-sites/java/docs/app-services/mongodb/watch.mdx:17]: i'm referring to the `.. important:: Severless Limitations` in all the SDK files. (also should be *serverless*

right now there are nested admonitions.

**Review by mongodben — COMMENTED** [review]
i noticed that the staging link, https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/DOCSP-23473/, isn't working

can you add a working link to the PR description

**Review by mongodben — APPROVED** [review]
lgtm
